proto/viam/app/data/v1/data.proto

```diff
@@ -188,7 +200,13 @@ message DeleteBinaryDataByIDsResponse {
 
 // Annotation specifies the ways in which we can associate annotations with data, e.g. with a text tag or image bounding box
 message Annotation {
-  string tag = 1;
+  string value = 3;
```

Comment:
numbering order here?

Reply:
Should be fixed with the change back to tags